### DISCUSSION TITLE
Unusual Message When Creating Virtual Environment

### DISCUSSION THREAD

**Discussion by deivguerrero** [pyenv/pyenv#3023] [Q&A]
Today, pyenv was updated to version 2.4.8. Everything seemed fine until I ran the command pyenv virtualenv VERSION name-env, which resulted in the following message:

> pyenv: no installed versions match the prefix `-q'

This issue didn't occur in version 2.4.7 (I switched versions using git checkout in the pyenv directory).

I am using MacOS 14.5 with an M2 chip.
Does anyone have any idea why this message appears?
(Attached is a screenshot)
<img width="870" alt="Screenshot 2024-07-21 at 14 25 48" src="https://github.com/user-attachments/assets/751f947b-9ae3-46fc-a0d4-1872518ca1ba">

**Comment by native-api** [comment]
https://github.com/pyenv/pyenv-virtualenv/issues/485
